src/python/pants/core/goals/test.py

```diff
@@ -500,6 +510,35 @@ async def run_tests(
     return Test(exit_code)
 
 
+def _format_test_summary(result: TestResult, run_id: RunId, console: Console) -> str:
+    """Format the test summary printed to the console."""
+
+    SourceMap = {
+        ProcessResultMetadata.Source.MEMOIZED: "memoized",
+        ProcessResultMetadata.Source.RAN_LOCALLY: "ran locally",
+        ProcessResultMetadata.Source.RAN_REMOTELY: "ran remotely",
+        ProcessResultMetadata.Source.HIT_LOCALLY: "cached locally",
+        ProcessResultMetadata.Source.HIT_REMOTELY: "cached remotely",
+    }
+
+    if result.exit_code == 0:
+        sigil = console.sigil_succeeded()
+        status = "succeeded"
+    else:
+        sigil = console.sigil_failed()
+        status = "failed"
+
+    source = result.result_metadata.source(run_id)
+    source_print = ""
+
+    if SourceMap.get(source):
+        source_print = f" ({SourceMap.get(source)})"
+
+    elapsed_time = result.result_metadata.total_elapsed_ms
+
+    return f"{sigil} {result.address} {status} in {elapsed_time} ms{source_print}."
```

Comment:
```suggestion
    elapsed_secs = result.result_metadata.total_elapsed_ms / 1000

    return f"{sigil} {result.address} {status} in {elapsed_secs:.2f}s {source_print}."
```

We have a nascent pattern of rendering floating point seconds with two decimal places (such as in the `--dynamic-ui`, and in logging), so let's do that here too.